README.md

```diff
@@ -49,15 +58,15 @@ In order to install SonarClojure:
     sonar.projectKey=your-project-key
     sonar.projectName=YourProjectName
     sonar.projectVersion=1.0
-    sonar.sources=src,project.clj
+    sonar.sources=src,test,resources,project.clj,target/nvd/dependency-check-report.json
```

Comment:
Do we need to include `resources` and `test` as part of the sources?

Reply:
I removed for while this but linters may analyze tests also. At least Kibit does this. If test directory is not in src then issue marking will fail. Also line calculation for cloverage needs tests also in sources. For resources it is not mandatory but maybe recommended because there might be xml files etc which should be scanned. Maybe this decision should be left to the user.